modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2605,6 +2631,7 @@ export class Wallet implements IWallet {
     let unsignedTx: EddsaUnsignedTransaction;
 
     if (txRequest.apiVersion === 'full') {
+      assert(txRequest.transactions);
```

Comment:
assert won't let the user what went wrong. Can we throw this with a useful message?

Or remove it and add a `?.` to line 2635 if that is all that is needed.